From: Jan Glauber <jang@linux.vnet.ibm.com>
Subject: s390/qdio: rename the misleading PCI flag of qdio devices
Patch-mainline: v3.8-rc1
Git-commit: 6726a807c38d7fd09bc23a0adc738efec6ff9492
References: bnc#848335,FATE#83037,LTC#94737

For each detected qdio device a line like to following is printed:
qdio: 0.0.4102 OSA on SC 1045 using AI:1 QEBSM:0 PCI:1 TDD:1 SIGA: W AP

The PCI flag is misleading as this stands for "program controlled interrupt".
Rename it to PRI "program requested interrupt" which is more accurate and
does not interfere with another popular piece of technology.

Leave the pci string in the code since changing that would result in a huge
patch.

Signed-off-by: Jan Glauber <jang@linux.vnet.ibm.com>
Signed-off-by: Martin Schwidefsky <schwidefsky@de.ibm.com>
Acked-by: John Jolly <jjolly@suse.de>
---
 drivers/s390/cio/qdio_setup.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/s390/cio/qdio_setup.c
+++ b/drivers/s390/cio/qdio_setup.c
@@ -486,7 +486,7 @@ void qdio_print_subchannel_info(struct q
 	char s[80];
 
 	snprintf(s, 80, "qdio: %s %s on SC %x using "
-		 "AI:%d QEBSM:%d PCI:%d TDD:%d SIGA:%s%s%s%s%s\n",
+		 "AI:%d QEBSM:%d PRI:%d TDD:%d SIGA:%s%s%s%s%s\n",
 		 dev_name(&cdev->dev),
 		 (irq_ptr->qib.qfmt == QDIO_QETH_QFMT) ? "OSA" :
 			((irq_ptr->qib.qfmt == QDIO_ZFCP_QFMT) ? "ZFCP" : "HS"),
